bash.exe.stackdump

```diff
@@ -0,0 +1,4 @@
+Stack trace:
```

Comment:
What is this?

Reply:
It just came up, maybe because I am using window.

Reply:
Put it .gitignore then, it's not project related.

Reply:
thanks just did.